docs/source/index.rst

```diff
@@ -1,6 +1,95 @@
-========================
-Scylla Monitoring Stack
-========================
+.. title:: Scylla Monitoring Stack
+
+.. raw:: html
+
+  <div class="landing landing--floating">
+
+
+.. hero-box::
+  :title: Scylla Monitoring Stack Documentation
+  :image: /_static/img/mascots/scylla-monitor.svg
+  :button_icon: fa fa-github
+  :button_url: https://github.com/scylladb/scylla-monitoring
+  :button_text: Scylla Monitoring Stack Project
+
+  Welcome to Scylla Monitoring Stack Documentation!
+  Scylla Monitoring Stack is a full stack for Scylla monitoring and alerting.
+  Older versions of Scylla Monitoring Stack Documentation are available `here <https://docs.scylladb.com/operating-scylla/monitoring/>`_.
```

Comment:
```suggestion
  A full stack for Scylla monitoring and alerting
```

Reply:
Making the subtitle shorter and to the point. No one care about the old version any more